components/raftstore/src/coprocessor/consistency_check.rs

```diff
@@ -60,12 +60,10 @@ impl<E: KvEngine> ConsistencyCheckObserver<E> for Raw<E> {
 fn compute_hash_on_raw<S: Snapshot>(region: &Region, snap: &S) -> Result<u32> {
     let region_id = region.get_id();
     let mut digest = crc32fast::Hasher::new();
-    let mut cf_names = snap.cf_names();
-    cf_names.sort_unstable();
 
     let start_key = keys::enc_start_key(region);
     let end_key = keys::enc_end_key(region);
-    for cf in cf_names {
+    for cf in ALL_CFS {
```

Comment:
If cf list is changed between versions, this check can fail. But it can also fail in the old implementation.